common/traits/wc_race_traits.txt

```diff
@@ -1904,6 +1904,26 @@ creature_stone_tolvir = {
 	}
 	shown_in_ruler_designer = no
 
+	inherit_chance = 50
+	both_parent_has_trait_inherit_chance = 100
+}
+creature_nightborne = {
```

Comment:
You haven't done steps to make it work. You have to...
Create a `portrait` graphical culture here.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/110262877-7520b380-7fce-11eb-95f0-3701d305b602.png)

</details>

Create a trigger here.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/110262992-cdf04c00-7fce-11eb-9f74-c0dcbe34426f.png)

</details>

Set the race trait here.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/110262937-97b2cc80-7fce-11eb-965f-b7b94fe5dbe5.png)

</details>

Check files for `creature_night_elf`. Since the nighrborne are indeed medium humanoids and elves.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/110262958-b022e700-7fce-11eb-80b6-5049832f0b71.png)
![image](https://user-images.githubusercontent.com/46384992/110262983-c630a780-7fce-11eb-8dd5-a6aef75f007a.png)

</details>

Reply:
done in 873fc010, in 0eaff8bd, in cc3fd423 and in d5b990ff

Reply:
You missed this part.

> Check files for `creature_night_elf`. Since the nighrborne are indeed medium humanoids and elves.

Nightborne still not mentioned in `copy_race_effect` and maybe more.